VS/VehicleSignals.rdf

```diff
@@ -873,6 +874,7 @@
 	
 	<owl:Class rdf:about="&auto-vs;CatalystTemperatureSensor">
 		<rdfs:subClassOf rdf:resource="&auto-vs;VehicleSensor"/>
+		<rdfs:label xml:lang="en">Catalyst Temperature Sensor</rdfs:label>
```

Comment:
Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.